contracts/src/v0.7/ArbitrumValidator.sol

```diff
@@ -0,0 +1,134 @@
+    uint256 gasCostL2;
+    address refundableAddress;
+  }
+  GasConfiguration private s_gasConfig;
+  uint32 constant private s_L2GasLimit = 30000000;
+  uint32 constant private s_maxSubmissionCostIncreaseRatio = 13;
+
+  /**
+   * @param aggregatorAddress default aggregator with access to validate
+   * @param inboxAddress address of the Arbitrum Inbox L1 contract
+   * @param flagAddress address of the Chainlink L2 Flags contract
+   * @param gasConfigAccessController address of the access controller for managing gas price on Arbitrum
+   * @param maxSubmissionCost maximum cost willing to pay on L2
+   * @param maximumGasPrice maximum gas price to pay on L2
+   * @param gasCostL2 value to send to L2 to cover gas fee
+   * @param refundableAddress address where gas excess on L2 will be sent
+   */
+  constructor(
+    address aggregatorAddress,
+    address inboxAddress,
+    address flagAddress,
+    address gasConfigAccessController,
+    uint256 maxSubmissionCost,
+    uint32 maximumGasPrice,
+    uint256 gasCostL2,
+    address refundableAddress
+  ) {
+    s_arbitrumInbox = ArbitrumInboxInterface(inboxAddress);
+    s_gasConfigAccessController = SimpleWriteAccessController(gasConfigAccessController);
+    s_flagsAddress = flagAddress;
+    _setGasConfiguration(maxSubmissionCost, maximumGasPrice, gasCostL2, refundableAddress);
+
+    SimpleWriteAccessController(address(this)).addAccess(aggregatorAddress);
+  }
+  
+  fallback() external payable {}
+
+  function withdrawFunds() 
+    external 
+    onlyOwner() 
+  {
+    address payable to = payable(msg.sender);
+    to.transfer(address(this).balance);
+  }
+
+  function withdrawFundsTo(
+    address payable to
+  ) 
+    external
+    onlyOwner() 
+  {
+    to.transfer(address(this).balance);
+  }
+
+  function setGasConfiguration(
+    uint256 maxSubmissionCost,
+    uint32 maximumGasPrice,
+    uint256 gasCostL2,
+    address refundableAddress
+  )
+    external
+  {
+    require(s_gasConfigAccessController.hasAccess(msg.sender, msg.data), "Only gas configuration admin can call");
+    _setGasConfiguration(maxSubmissionCost, maximumGasPrice, gasCostL2, refundableAddress);
+  }
+
+  function validate(
+    uint256 /* previousRoundId */,
+    int256 /* previousAnswer */,
+    uint256 /* currentRoundId */,
+    int256 currentAnswer
+  ) 
+    external 
+    override
+    checkAccess() 
+    returns (bool) 
+  {
+    bytes memory data = currentAnswer == 1 ? abi.encodeWithSelector(RAISE_SELECTOR, s_arbitrumFlag) : abi.encodeWithSelector(LOWER_SELECTOR, [s_arbitrumFlag]);
+
+    s_arbitrumInbox.createRetryableTicket{value: s_gasConfig.gasCostL2}(
+      s_flagsAddress, 
+      0, // L2 call value
+      s_gasConfig.maximumSubmissionCost, // Max submission cost of sending data length
+      s_gasConfig.refundableAddress, // excessFeeRefundAddress
+      s_gasConfig.refundableAddress, // callValueRefundAddress
+      s_L2GasLimit,
+      s_gasConfig.maximumGasPrice, 
+      data
+    );
```

Comment:
Unused return value

Reply:
Right, just there cause the interface expects the return